Makefile

```diff
@@ -83,7 +83,7 @@ clean: $(SUBDIRSCLEAN)
 	rm -f *~ */*~ */*/*~
 
 $(SUBDIRSCLEAN): %.clean:
-	$(MAKE) -C $* clean
+	[ ! -e $* ] || $(MAKE) -C $* clean
```

Comment:
This is if you download only the `-src-` tar gz, right? Otherwise, all dirs are expected to be in place.

Reply:
I already merged this